Source/animationinfos.cpp

```diff
@@ -11,35 +11,32 @@
 
 namespace devilution {
 
-int AnimationInfos::GetFrameToUseForRendering()
+int AnimationInfos::GetFrameToUseForRendering() const
```

Comment:
This should be done in the commit where the function is moved.

Reply:
I restrained the moved-commits, so they don't include anything expect the move.
And const is a small change, but a change.
So I hope it's okay if I don't rewrite the history here. 🙂

Reply:
> And const is a small change, but a change.

The need for const was the move, so they should be done in the same change.

> So I hope it's okay if I don't rewrite the history here. 🙂

You'll have to anyways to fix the other issues.

Reply:
Do you want me to rewrite the history for every change? 🤔 
I would append them as new commits, so they are easier to review and you don't have to look at all changes again.
If you like to squash this pr. I have no problem with that.

Reply:
> Do you want me to rewrite the history for every change? 🤔

That's the best way to preserve a clean history, so yes.

> I would append them as new commits, so they are easier to review and you don't have to look at all changes again.

That would completely defeat three purpose of your split.

> If you like to squash this pr. I have no problem with that.

That's up to Ajenbo. I'd only squash if the history was messed up, which is not the case here. The way you split the changes is not only helpful for reviewing purposes, but for checking the history later too.

Reply:
You can also prefix the commits with "fixup" then they will be combined when merging.

Reply:
Nice. 👍  Didn't know about --fixup.
I'm still learning to work with git.
In the next fixup I will try it out. Thanks 🙂

Reply:
> You can also prefix the commits with "fixup" then they will be combined when merging.

What?! I've never heard of this as well! Is that a feature of Github specifically @AJenbo , or does it work on raw git too?

Reply:
Don't have anything to do with GitHub. I thought I already mentioned it to you in the chat :)

To apply the fixups you just run:
`git rebase origin/master --autosquash -i`

Reply:
I use fixup all the time while rebasing, but never knew you could also use by modifying the commits themselves. Very cool.